### PR TITLE
sdn: add better logging of ofport request failure

### PR DESCRIPTION
**Pull request by dcbw**
So we can figure out what the actual error is.

Related: https://bugzilla.redhat.com/show_bug.cgi?id=1467387

@openshift/networking @knobunc

### CODE REVIEW

**Comment by knobunc** [comment]
[test]

**Comment by knobunc** [comment]
[merge] This can't break anything, and the improved logging may prove useful.

File: pkg/util/ovs/ovs.go [hunk 1]
```diff
@@ -199,7 +199,7 @@ func (ovsif *ovsExec) AddPort(port string, ofportRequest int, properties ...stri
 	}
 	ofport, err := ovsif.GetOFPort(port)
 	if ofportRequest > 0 && ofportRequest != ofport {
-		return -1, fmt.Errorf("Allocated ofport (%d) did not match request (%d)", ofport, ofportRequest)
+		return -1, fmt.Errorf("Allocated ofport (%d) did not match request (%d): %v", ofport, ofportRequest, err)
```
Comment (danwinship) [inline pkg/util/ovs/ovs.go]: we should probably have a separate message for the err!=nil case; this error message was basically written under the assumption that GetOFPort will always succeed

Comment (danwinship) [inline pkg/util/ovs/ovs.go]: oh, also, error messages are supposed to start with lowercase. maybe fix that both here and in GetOFPort

Comment (dcbw) [inline pkg/util/ovs/ovs.go]: @danwinship fixed

**Comment by knobunc** [comment]
[merge][severity: blocker] This is just logging changes to help diagnose a problem.  It should be perfectly safe.

**Pull request merged**